### PR TITLE
Ds starting on UI

### PR DESCRIPTION
**Pull request by dylansarikas**
Creates initial UI and Bootstrap Navbar

### CODE REVIEW

**Review by APatel-AI — APPROVED** [review]
Great Job Dylan!!.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (JoseMDO) [inline app/controllers/application_controller.rb:2]: Nice!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,14 +6,26 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:9]: Great job adding your cdn assets to a partial! This helps keep the layout file clean

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -6,14 +6,26 @@
...
+
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
 
   <body>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
+    <%= render partial: "shared/navbar" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:16]: same as above

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -6,14 +6,26 @@
...
+
+    <div class="container">
+      <% if notice.present? %>
+        <%= render partial: "shared/flash", locals: { message: notice, css_class: "success" } %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:20]: same as above

File: app/views/shared/_navbar.html.erb [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <ul class="dropdown-menu">
+            <a class="dropdown-item" href="/photos">Photos</a>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:16]: remember we can start using helper methods like link_to here

File: lib/tasks/dev.rake [hunk 6]
```diff
@@ -11,15 +11,22 @@ task :sample_data => :environment do
...
+
   10.times do
-    name =  Faker::Name.first_name
-    user = User.create(
-      username: name,
-      email: name.downcase + "@email.com",
+    usernames << Faker::Name.first_name
+  end
+
+  usernames.each do |username|
```
Comment (JoseMDO) [inline lib/tasks/dev.rake:23]: nice!

**Review by JoseMDO — APPROVED** [review]
Everything looks good Dylan!

**Pull request merged**